docs/api-reference/next/image.md

```diff
@@ -114,14 +84,16 @@ The layout behavior of the image as the viewport changes size.
 
 ### loader
 
-A custom function used to resolve URLs. Defaults to [`images` object in `next.config.js`](/docs/basic-features/image-optimization.md#loader).
+A custom function used to resolve URLs. Setting the loader as a prop on the Image component overrides the default loader defined in the [`images` section of `next.config.js`](#set-default-loader).
 
-`loader` is a function returning a string, given the following parameters:
+Defining a custom `loader` is simple. A loader function returns a URL string for the image, given the following parameters:
```

Comment:
```suggestion
A `loader` is a function returning a URL string for the image, given the following parameters:
```

I try to stay away from words like "simple", because users might have a tough time and it would not feel simple 😄